projects/plugins/jetpack/extensions/blocks/contact-info/email/save.js

```diff
@@ -1,6 +1,6 @@
 import { useBlockProps } from '@wordpress/block-editor';
 import { Fragment } from '@wordpress/element';
-import emailValidator from 'email-validator';
+import * as emailValidator from 'email-validator';
```

Comment:
```suggestion
import { validate } from 'email-validator';
```

And `emailValidator.validate` → `validate` on line 9?